packages/code-du-travail-frontend/src/modules/enterprise/EnterpriseAgreementSearch/EnterpriseAgreementSelection.tsx

```diff
@@ -0,0 +1,121 @@
+import { fr } from "@codegouvfr/react-dsfr";
+import Card from "@codegouvfr/react-dsfr/Card";
+import Button from "@codegouvfr/react-dsfr/Button";
+import { Enterprise } from "../types";
+import { ButtonStyle, CardTitleStyle } from "../../convention-collective/style";
+import { css } from "../../../../styled-system/css";
+
+type Props = {
+  enterprise: Omit<Enterprise, "complements">;
+  widgetMode?: boolean;
+};
+
+export const EnterpriseAgreementSelection = ({
+  enterprise,
+  widgetMode = false,
+}: Props) => {
+  return (
+    <>
+      <p className={fr.cx("fr-h4", "fr-mt-2w", "fr-mb-0")}>
+        {enterprise.conventions.length === 0
+          ? `Aucune convention collective n'a été déclarée pour l'entreprise`
+          : enterprise.conventions.length === 1
+            ? `1 convention collective trouvée pour :`
+            : `${enterprise.conventions.length} conventions collectives trouvées pour :`}
+      </p>
+      <p className={fr.cx("fr-text--bold", "fr-m-0", "fr-mt-2w")}>
+        {enterprise.label}
+      </p>
+      <p className={fr.cx("fr-m-0")}>
+        Activité : {enterprise.activitePrincipale}
+      </p>
+      <p className={fr.cx("fr-mb-2w")}>{enterprise.address}</p>
+      {enterprise.conventions?.map((agreement) => {
+        const disabled =
+          !agreement.slug || !agreement.url || !agreement.contributions;
+        let description;
+        if (!agreement.slug) {
+          description =
+            "Nous n’avons pas d’informations concernant cette convention collective";
+        } else if (agreement.url || agreement.contributions) {
```

Comment:
Il manque le cas ou on a un slug j'ai l'impression mais je suis pas sure ?
```suggestion
        } else if (agreement.url || agreement.contributions || agreement.slug) {
```

Reply:
ah oui

Reply:
il faudrait rajouter un test